Reyes_JohnRuiz/index.html

```diff
@@ -91,39 +91,44 @@ <h2>My Goals</h2>
         <ol>
           <li>To be a Developer</li>
           <li>To learn some new Skills</li>
-          <li>to travel outside of country </li>
-          <li>to </li>
-          <ol>To be happy</ol>
+          <li>to travel outside of country</li>
+          <li>to</li>
+          <ol>
+            To be happy
+          </ol>
         </ol>
       </section>
 
       <section>
         <h2>Add a comment here</h2>
-        <form id="comment-form" class="add-comment">
+        <form id="comment_form" class="add-comment">
           <input
             type="text"
             name="name2"
-            id="comment-name"
+            id="comment_name"
             placeholder="Enter your name"
           />
           <textarea
```

Comment:
Don't separate the closing tag in a new line. 
```suggestion
            placeholder="Enter your name"/>
          <textarea
```